lib/web_ui/lib/src/engine/keyboard_binding.dart

```diff
@@ -211,10 +228,28 @@ class FlutterHtmlKeyboardEvent {
 // [dispatchKeyData] as given in the constructor. Some key data might be
 // dispatched asynchronously.
 class KeyboardConverter {
-  KeyboardConverter(this.performDispatchKeyData, {this.onMacOs = false});
+  KeyboardConverter(this.performDispatchKeyData, OperatingSystem platform)
+    : onMacOs = platform == OperatingSystem.macOs,
+      _mapping = _mappingFromPlatform(platform);
 
   final DispatchKeyData performDispatchKeyData;
+  // Whether the current platform is macOS, which affects how certain key events
```

Comment:
Is it the case that `onMacOS` is *only* used to differentiate things for macOS, or are these keyboard quirks also present on iOS?

Reply:
Nice find. I've tested it on my iPad and I confirm that they also present on iOS. I'm changing it to onDarwin.

Reply:
Let me make it a separate PR after this PR.